components/finance/FinanceCategoriesListWrapper/index.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+function FinanceCategoriesListWrapper({ children }: any) {
```

Comment:
this component does not serve any purpose. Have the <ul> inline in the parent instead

Reply:
fixed